test/TensorFlowRuntime/dynamic_compilation_tensor_group.swift

```diff
@@ -112,6 +114,26 @@ TensorGroupTest.testAllBackends("input, loadable") {
   expectEqual(ShapedArray(shape: [2, 2], scalars: [1, 2, 3, 4]), packed.array)
 }
 
+@inline(never)
+func some_tf_op(n : Int) {
+  var arr: [Tensor<Float>] = []
+  var arr_exp: [Float] = []
+  for i in 0..<n {
+    arr.append(Tensor<Float>([Float(i), 2, 3]))
+    arr_exp.insert(contentsOf: [Float(i), 2, 3], at: arr_exp.count)
+  }
+
+  let ret: TensorHandle<Float> = #tfop("Pack",
```

Comment:
You should be able to do `let ret: Tensor<Float>` to avoid having to construct the Tensor yourself. This may require my very latest change that I submitted just about an hour ago.